samples/cards/README.md

```diff
@@ -0,0 +1,20 @@
+# Sample: cards
+
+a demo of adaptive cards
+
+## Prerequisites
+
+- Node.js version 18 or later
+- An Microsoft 365 development account. If you don't have one, you can get one for free by signing up for the [Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program).
```

Comment:
```suggestion
- A Microsoft 365 development account. If you don't have one, you can get one for free by signing up for the [Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program).
```

Reply:
removed this, keeping these readme's light